lib/container_server.go

```diff
@@ -302,7 +302,9 @@ func (c *ContainerServer) Update() error {
 		}
 		sb.RemoveInfraContainer()
 		c.ReleasePodName(sb.Name())
-		c.RemoveSandbox(sb.ID())
+		if err := c.RemoveSandbox(sb.ID()); err != nil {
```

Comment:
on an error now we skip `c.podIDIndex.Delete(sb.ID())`.  Is it the desired effect?

Reply:
No, switched to an debug warning if the removal fails.